js2xliff.js

```diff
@@ -1,61 +1,46 @@
-const xml2js = require('xml2js');
+const convert = require('xml-js');
+const {makeElement, makeText} = require('./util/makeNodes');
 
 function js2xliff(obj, opt, cb) {
 
   if (typeof opt === 'function') {
     cb = opt;
-    opt = { pretty: true, indent: '  ', newline: '\n' };
+    opt = { indent: '  ' };
```

Comment:
This is the only api difference that is caused by switching XML parsing libraries. `xml2js` supports the `pretty, `indent`, and `newline` options, but `xml-js` only exposes `spaces`, which is functionally like `indent` and `pretty` combined. If `spaces` is `0` then that's the same as `pretty: false`. Otherwise spaces can be a number (the number of spaces) or actual characters (as shown here, using two space characters).